force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +221,20 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        public String title;
+        public String author;
+        public Integer pages;
     
         // Book constructor to initialize the title and author
-        public Book() {
-            // Initialize the title and author
+        public Book(String title, String author) {
+            this.title = title;                     // Initialize the title and author
+            this.author = author;
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
-            // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            String bookDetails = 'Title: ' + title +', Author: ' +author;
```

Comment:
While not absolutely necessary in Apex, it's best to reference instance variables using `this`:
```suggestion
            String bookDetails = 'Title: ' + this.title +', Author: ' + this.author;
```